### PR TITLE
Fix storybook prop validation warnings

### LINKED ISSUE #4083: Fix Storybook prop validation warnings
**Is your feature request related to a problem? Please describe.**
In Psammead Storybook, some components throw prop validation fails in console, i.e. `psammead-media-player` (although there may be more). This litters the terminal and makes it less likely that we'll catch _actual_ bugs

```
    Warning: Failed prop type: The prop `noJsMessage` is marked as required in `CanonicalMediaPlayerComponent`, but its value is `undefined`.
    
    Warning: Failed prop type: The prop `noJsMessage` is marked as required in `Placeholder`, but its value is `undefined`.
    
    Warning: Failed prop type: The prop `noJsMessage` is marked as required in `Guidance`, but its value is `undefined`.
```

**Describe the solution you'd like**
- Check Storybook for any such prop validation errors, and fix them by making sure that the component is receiving the necessary props.

**Testing notes**
[Tester to complete]

Dev insight: Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by akeelm**
Resolves #4083

**Overall change:** Fixing the prop type console warnings for stories

**Code changes:**

- In packages/components/psammead-content-anchor/src/index.stories.jsx - the Headline component had warnings for missing script and service props. I found examples where "latin" is passed from gel-foundations/scripts into the script prop and "news" seems to be set as a default service for many of the components. 
- In packages/components/psammead-media-player/src/index.stories.jsx - the noJsMessage prop was missing for many of the components. I used the pidgin example that was already being used elsewhere in the file. I also noticed that the "AMP" player wasn't displaying due to a content warning, so I updated the URL to a working one - which was being used by the component "Audio Skin AMP" further down. 
- In packages/components/psammead-navigation/src/index.stories.jsx - "Dropdown animation" was missing the various colour props.

**Additional info:**
I'm a first time contributor to this repo. I've read through the guidelines and checked some existing PR's to ensure I'm not missing any specific formatting of commit messages etc. 

These changes haven't required any tests to be updated.

---

- [ ] (BBC contributors only) This PR follows the [repository use guidelines](https://github.com/bbc/simorgh-infrastructure/blob/latest/documentation/repository-guidelines.md)
- [ ] I have assigned myself to this PR and the corresponding issues
- [ ] Automated jest tests added (for new features) or updated (for existing features)
- [ ] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-content-anchor/src/index.stories.jsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { withKnobs } from '@storybook/addon-knobs';
 import Paragraph from '@bbc/psammead-paragraph';
 import { Headline } from '@bbc/psammead-headings';
 import { withServicesKnob } from '@bbc/psammead-storybook-helpers';
+import { latin } from '@bbc/gel-foundations/scripts';
 import notes from '../README.md';
```
Comment (ryanmccombe) [inline packages/components/psammead-content-anchor/src/index.stories.jsx]: `latin` import will be unnecessary if later suggestions applied
```suggestion
import notes from '../README.md';
```

File: packages/components/psammead-content-anchor/src/index.stories.jsx [hunk 2]
```diff
@@ -90,7 +91,9 @@ storiesOf('Components/ContentAnchor', module)
     ({ text, script, service }) => {
       return (
         <Wrapper>
-          <Headline>{text}</Headline>
+          <Headline script={latin} service="news">
```
Comment (ryanmccombe) [inline packages/components/psammead-content-anchor/src/index.stories.jsx]: `script` and `service` are available as parameters in this function 😄 
```suggestion
          <Headline script={script} service={service}>
```

File: packages/components/psammead-content-anchor/src/index.stories.jsx [hunk 3]
```diff
@@ -104,7 +107,9 @@ storiesOf('Components/ContentAnchor', module)
...
```
Comment (ryanmccombe) [inline packages/components/psammead-content-anchor/src/index.stories.jsx]:
```suggestion
          <Headline script={script} service={service}>
```

File: packages/components/psammead-content-anchor/src/index.stories.jsx [hunk 4]
```diff
@@ -123,7 +128,9 @@ storiesOf('Components/ContentAnchor', module)
...
```
Comment (ryanmccombe) [inline packages/components/psammead-content-anchor/src/index.stories.jsx]:
```suggestion
          <Headline script={script} service={service}>
```

**Review by ryanmccombe — APPROVED** [review]
Thanks for the PR! 👍

**Review by andrewscfc — APPROVED** [review]
Thanks for picking this up, we really appreciate it 🙂

**Comment by HarveyPeachey** [comment]
Hi, @akeelm thanks for your contribution! Could you do us a favour and sync your [fork](https://github.com/akeelm/psammead) so it pulls in the latest version of psammead? We want to check if it fixes a GitHub action issue we're having

**Comment by akeelm** [comment]
> Hi, @akeelm thanks for your contribution! Could you do us a favour and sync your [fork](https://github.com/akeelm/psammead) so it pulls in the latest version of psammead? We want to check if it fixes a GitHub action issue we're having

Hey @HarveyPeachey, have sync'd my branch with the latest!

**Comment by paruchurisilpa** [comment]
Moving it to done as it doesn't need any manual testing..

**Pull request merged**